docs/config_samples/dataset_types/ga_s2_ard.yaml

```diff
@@ -13,18 +13,19 @@ measurements:
 - name: 'pixel_quality'
   aliases: [mask, CFmask]
   dtype: uint8
-  nodata: 255
+  nodata: 0
   units: '1'
   flags_definition: 
     cfmask:
       bits: [0,1,2,3,4,5,6,7]
       description: CFmask
```

Comment:
CFmask should be Fmask. The C version of Fmask was not used in the product creation.